### DISCUSSION TITLE
Table Headers

### DISCUSSION THREAD

**Discussion by steve-alt-ctrl** [filamentphp/filament#6014] [Help] [answered]
Forgive me if this was asked already but I didn't see this in the documentation.  How would I go about updating the table header within a resource?

**Comment by steve-alt-ctrl** [comment, marked as answer]
I figured this one out.  Leaving this here for others.

```
protected function getTableHeading(): ?string
{
    return 'My New Heading';
}
```

**Reply by hammadzafar05** [reply]
Thanks for sharing here,
kindly add it in the documentation as well.
